Documentation/spec/docfx_flavored_markdown.md

```diff
@@ -111,7 +111,7 @@ Allows you to insert code with code language specified. The content of specified
 * __`<codepath>`__ is the relative path in file system which indicates the code snippet file that you want to expand.
 * __`<queryoption>`__ and __`<queryoptionvalue>`__ are used together to retrieve part of the code snippet file in the line range or tag name way. We have 2 query string options to represent these two ways:
```

Comment:
2 option: `#` and `?`, and option`?` has 3 sub options ....
I think it's more clear if I use a table.